### ISSUE TITLE
Visual Studio 2019 Support

### ISSUE THREAD

**Issue by Jamizzle93** [justcla/KeyBindings#14]
Firstly, I'd like to say thank you for making such a great extension.
I love being able to use IntelliJ keyboard shortcuts in Visual Studio and it makes me much more productive, so thanks for that!

I was just wondering if there is any plan to support Visual Studio 2019?

Once again, thanks for developing such a great extension.

**Comment by pishangujeniya** [comment]
I want to have it in my Visual Studio 2019.

**Comment by KKowalewski24** [comment]
It would be really nice if this magic extension could be available for Visual Studio 2019.

**Comment by justcla** [comment]
Gotcha. I'll try to do it tonight. ☺️

**Comment by justcla** [comment]
Now that v16.1 includes toggle line comment and toggle block comment, I should probably update some of the key maps.

**Comment by hkuadithya** [comment]
@justcla could you please post an update on this thread once you've published the tool on Marketplace?

In addition, the VS code has a large number of popular extensions for IntelliJ keybindings. However, for Visual Studio, I could only find your extension in the marketplace.  I wonder why there aren't other popular keybinding tools for Visual Studio apart from yours. 

Since you were a manager at Microsoft for the VS IDE team, I think you are in a very good position to answer this question.

**Comment by justcla** [comment]
Hi Adithya. Absolutely, I'll post updates here.

Update: I had a look at it last night. There are two extensions: One with all the keyboard shortcut mapping schemes, and one with the keyboard shortcut management tools (load, save, import, etc).
I need to upgrade the Keyboard Shortcuts Manager first, then I can embed it into the mapping scheme extension.

Question for you (and others): What are you looking for most? The mapping schemes I developed or the management tools (Tools->Keyboard Shortcuts)?
(FYI. If you just want the mapping schemes, you can grab them directly from [this repo](https://github.com/justcla/KeyBindings/tree/master/KeyboardSchemes/VSKs/2017) and drop them in your Common7/IDE directly.)

> In addition, the VS code has a large number of popular extensions for IntelliJ keybindings. However, for Visual Studio, I could only find your extension in the marketplace. I wonder why there aren't other popular keybinding tools for Visual Studio apart from yours.

I guess I was the only one who cared.

**Comment by justcla** [comment]
OK. I've upgraded the Keyboard Shortcuts Manager. It is available [here](https://marketplace.visualstudio.com/items?itemName=JustinClareburtMSFT.VSShortcutsManager).

Note: It is a MAJOR upgrade and includes some very powerful shortcut management features that were developed at the Microsoft Hackathon 2018. Not everything works perfectly, but it's pretty useful.

Now I can look at upgrading the HotKeys extension.

**Comment by justcla** [comment]
Sorry for the delay. I have now released the upgraded HotKeys VSIX for VS2019. :-D

Importantly, it includes the upgraded and improved shortcuts manager v2.1.0.

Note that I did not include and VSKs this time. Instead I included the VSSettings files, which users can "Load" from the Keyboard shortcuts menu. I found that's the cleanest way to apply shortcuts and it also lets you apply one set over another. I use the "Reset" in between, as necessary.

Anyway, I hope you like it! I've been using the IntelliJ bindings on VS2019 this week and find it much more useful and productive. :-)
